packages/adapter-node/index.js

```diff
@@ -1,6 +1,9 @@
 import { readFileSync, writeFileSync } from 'fs';
 import { fileURLToPath } from 'url';
-import * as esbuild from 'esbuild';
+import { rollup } from 'rollup';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
```

Comment:
do we need the JSON plugin? will people be importing JSON that hasn't already been bundled by Vite?

Reply:
They could be using a devdependency that includes JSON which isn't bundled yet but which we want to bundle when adapting.

Reply:
yep, this needs to work with `require('./stuff.json')`